worlds/witness/options.py

```diff
@@ -227,6 +227,24 @@ class HintAmount(Range):
     default = 12
 
 
+class VagueHints(Choice):
+    """Make Location Hints a bit more vague, where they only tell you about the general area the item is in.
+    Area Hints will be generated as normal.
+
+    If set to "stable", only location groups will be used. If a game one of your items ended up in doesn't implement location groups, your hint will instead only tell you that the item is "somewhere in" that game.

```

Comment:
```suggestion
    If set to "stable", only location groups will be used. If location groups aren't implemented for the game your item ended up in, your hint will instead only tell you that the item is "somewhere in" that game.
```
I don't have a good technical explanation for this, but I think it's easier to follow.